ramls/patron-action-session.raml

```diff
@@ -47,3 +48,39 @@ resourceTypes:
           exampleItem: !include examples/patron-action-session.json
       put:
         is: [validate]
+  /expired-session-patron-ids:
+      get:
+        queryParameters:
+          action_type:
+            description: Action type for patron notice session
+            type: string
+            enum: ["Check-in", "Check-out"]
+            required: true
+          last_time_action_limit:
```

Comment:
Why a separate parameter was introduced? 
This value is pre-configured and is it it possible to retrieve value internally?

Reply:
Do you mean retrieve from somewhere like `mod-configuration`? If so, we shouldn't call other modules from `storage` modules, AFAIR.

Reply:
correct